include/exadg/operators/integrator_flags.h

```diff
@@ -55,41 +54,11 @@ struct CellFlags
   dealii::EvaluationFlags::EvaluationFlags flags{dealii::EvaluationFlags::nothing};
 };
 
-struct FaceFlags
-{
-  explicit FaceFlags(
-    dealii::EvaluationFlags::EvaluationFlags const flags_in = dealii::EvaluationFlags::nothing)
-    : flags(flags_in){};
-
-  FaceFlags
-  operator|(FaceFlags const & other)
-  {
-    FaceFlags face_flags_combined;
+using FaceFlags = CellFlags;
```

Comment:
I think the reason for distinguishing cell and face flags before was that we do not have second derivatives for faces. However, this might no longer be up-to-date on the deal.II side. So deal.II does not distinguish between cell and face flags at all, is that correct?

If we decide to eliminate face flags, I agree to change the name `CellFlags`. However, I would suggest ``IntegratorFlags``  because we use the name `Integrator` instead of `FEEvaluation` in ExaDG. On the other hand, we have places where the ExaDG flags and the deal.II flags come into touch, where reading the code (`IntegratorFlags` and `dealii::EvaluationFlags` in the same line) might look strange?

Reply:
ah, this suggestion does not make sense. We already have `IntegratorFlags`. So we would follow your suggestion.

Reply:
Yes, I think the differentiation is obsolete, since there is a function called ```get_hessians()``` in the ```FEFaceEvaluation``` class. Anyway, deal.II would have to deal with that since ```dealii::EvaluationFlags``` is used in both cases.

We could also go with ```IntegratorFlagCollection``` instead of ```IntegratorFlags```, and ```IntegratorFlags``` instead of ```CellFlags```

Reply:
https://github.com/exadg/exadg/pull/243#discussion_r915722844
If we do it with bitwise operations everywhere, we can get rid of CellFlags completely. If you don't have a strong opinion on this, I think I would go with this option, which also frees us from the above problems with the naming of the classes.

Reply:
0a2b9b022e56932f14a8e72e7963f3c7af0fcdd0